apps/src/templates/FinishDialog.jsx

```diff
@@ -1,3 +1,5 @@
+/* global dashboard */
```

Comment:
Choosing this approach over importing `project.js` to avoid the circular dependency.